plugins/services/src/js/utils/__tests__/CreateServiceModalFormUtil-test.js

```diff
@@ -1,6 +1,27 @@
 const CreateServiceModalFormUtil = require('../CreateServiceModalFormUtil');
 
+const EMPTY_TYPES = [null, undefined, {}, [], ''];
```

Comment:
As a matter of fact `0` will be removed. If this is not considered an empty type I will have to fix this.